addons/missileguidance/functions/fnc_seekerType_MCLOS.sqf

```diff
@@ -0,0 +1,32 @@
+#include "..\script_component.hpp"
+/*
+ * Author: tcvm
+ * SACLOS seeker
+ *
+ * Arguments:
+ * 1: Guidance Arg Array <ARRAY>
+ * 2: Seeker State <ARRAY>
+ *
+ * Return Value:
+ * Position of wanted missile pos relative to the camera direction <ARRAY>
+ *
+ * Example:
+ * [] call ace_missileguidance_fnc_seekerType_SACLOS
+ *
+ * Public: No
+ */
+params ["", "_args"];
+_args params ["_firedEH", "", "_flightParams"];
+_firedEH params ["","","","","","","_projectile","_gunner"];
+_flightParams params ["_pitchRate", "_yawRate"];
+
+private _projectilePos = getPosASLVisual _projectile;
+
+private _accelerationDirection = _projectile vectorModelToWorldVisual (_gunner getVariable [QGVAR(MCLOS_direction), [0, 0, 0]]);
```

Comment:
1) I have the impression that this doesn't account for a missile being fired when a previous one is still flying. As far as I can tell, this would apply the direction to both missiles, when it should only apply it to the last one being fired (as far as I'm aware).
2) The direction (`QGVAR(MCLOS_direction)`) doesn't reset when a new missile is fired. Is that intentional? See https://github.com/acemod/ACE3/pull/6996/files#diff-4bd7ce264181ac0fab308b36a498cf7bbb168209494de181f8dc40e4fb5ad8f5R26 for comparison.

Reply:
1 - if you shoot two missiles from one joystick, you will control two missiles. The Malyutka ATGM can have up to 4 launchers attached to one joystick, and they would all follow the inputs iirc
2 - if the player is holding an input when they fire a new missile, the missile will follow the new input. since on key up the vector is reversed, utlimately the input will null out